### PR TITLE
Remove tui-dom 3.0 dependency on example html files (fix #561) and Apply Github action for checking error example page

### LINKED ISSUE #561: Clarify dependency on tui.dom/v3.0.0
README.md contains:
```
## 🔩 Dependency

* [tui-code-snippet](https://github.com/nhn/tui.code-snippet) >= 1.5.0  
* [tui-date-picker](https://github.com/nhn/tui.date-picker) >= 4.0.2 is optional.
* [tui-time-picker](https://github.com/nhn/tui.time-picker) >= 2.0.1 is optional.
```
• Why do the examples contain ` <script src="https://uicdn.toast.com/tui.dom/v3.0.0/tui-dom.js"></script>`, if it is not a dependency?
• Write < 2 for tui-code-snippet, to clarify that the newer code-snippet are not compatible

### PR DESCRIPTION
**Pull request by jungeun-cho**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* 1f15cfc : chore: remove tui-dom 3.0 dependency (fix #561)
* 522b5a2 : chore: apply github actions for checking error page automatically

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by jungeun-cho** [comment]
ok to test

File: createConfigVariable.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+function getGlobalVariable() {
+    if (!globalErrorLogVariable) {
+        throw Error("not exist examples path at tuidoc.config.json");
+    }
+
+    fs.writeFileSync("errorVariable.txt", globalErrorLogVariable);
+}
+
+getTestUrls();
+getGlobalVariable();
```
Comment (seonim-ryu) [inline createConfigVariable.js:36]: 이 파일은 린트가 동작을 안하는 것 같은데, `''` 따옴표 같은 컨벤션은 맞춰주는게 좋을 것 같습니다ㅎ
```js
fs.writeFileSync('errorVariable.txt', globalErrorLogVariable);
```

File: tuidoc.config.json [hunk 2]
```diff
@@ -48,7 +48,8 @@
       "example14-template-for-month": "Template for Month",
       "example15-template-for-timegrid": "Template for Week, Day",
       "example16-template-for-popup": "Template for Popup"
-    }
+    },
+    "globalErrorLogVariable": true
```
Comment (seonim-ryu) [inline tuidoc.config.json:52]: 요건 `@toast-ui/doc` 배포 후에 한 번 더 확인해야겠네요ㅎ

File: createConfigVariable.js (hunk 1 above)
Comment (jungeun-cho) [reply createConfigVariable.js:37]: 린트룰 적용하여 수정하였습니다.

File: tuidoc.config.json (hunk 2 above)
Comment (jungeun-cho) [reply tuidoc.config.json:52]: 네, doc 배포되고 실제 액션 테스트 예정입니다.

**Comment by jungeun-cho** [comment]
@seonim-ryu 
네, 코드스니펫 버전 1.x를 사용하고있으며, 캘린더 2.0에서 코드 스니펫의 메이져 버전을 올렸습니다.ㅎ
리뷰 남겨주신 내용 반영했습니다. 린트룰 반영했구요ㅎ

**Comment by jungeun-cho** [comment]
ok to test

**Pull request merged**